src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/features/form/validation/singleField/singleFieldValidationSagas.ts

```diff
@@ -17,28 +18,31 @@ export function* runSingleFieldValidationSaga(): SagaIterator {
     state.instanceData.instance,
   );
   const url = getDataValidationUrl(state.instanceData.instance.id, currentTaskDataId);
+  const currentSingleFieldValidation = state.formValidations.currentSingleFieldValidation;
```

Comment:
why not use `const {currentSingleFieldValidation} = state.formValidations;` if not referring with a shorter constant name?